thin_client/test/test_networking.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import thin_client.networking as networking
+import socket
+
+class test_networking(unittest.TestCase):
```

Comment:
PascalCase for class name. `ProtocolTest`, and rename file name to `protocol_test.py`